src/vision/Offline_Filter.py

```diff
@@ -1,141 +1,154 @@
-# PyLint please stop yelling at me I did nothing wrong...
-from math import radians, degrees
+# Final FRC 2020 vision program
```

Comment:
How is this different from the "final" file? If it does the same logic, it should be moved into a function and then into a shared file.